### ISSUE TITLE
Lyric parsing issue (3.1.7)

### ISSUE THREAD

**Issue by awyeamrkrabs** [surfbryce/beautiful-lyrics#83]
Hey, it seems that the issue is fixed in the song in the example, but it still happens on another song..

This song is called Ingkar by Tulus
![image](https://github.com/surfbryce/beautiful-lyrics/assets/58422403/6c39b4fe-fe0e-4210-9c13-940c2222ce28)
Where it should be like this:
![image](https://github.com/surfbryce/beautiful-lyrics/assets/58422403/5ae3e362-855c-4b09-b3ed-c587820e7c2a)

And there are also some songs, in which the lyric is displayed correctly, but right-aligned. In this case sometimes, when there are special characters like '?', then that special character would be misplaced. 

For example in this song (BARU by GAC)
![image](https://github.com/surfbryce/beautiful-lyrics/assets/58422403/354d2ff5-bfdb-40f4-86ce-3f66cfabe757)

And there is also an instance where there are some words that are displayed correctly, and some that are displayed right-to-left.

For example in this song (Badai Telah Berlalu by Diskoria, Laleilmanino, Bunga Citra Lestari)
![image](https://github.com/surfbryce/beautiful-lyrics/assets/58422403/0f9a96ed-bb01-4060-8641-1e1acdb64489)
Where it should be like this
![image](https://github.com/surfbryce/beautiful-lyrics/assets/58422403/f09eb91d-dae3-4bef-b018-fde2c6a63dc7)
Here, the words such as "angin", "malam", "kapan", "kau", "ku" are displayed correctly.

Thank you:)

_Originally posted by @awyeamrkrabs in https://github.com/surfbryce/beautiful-lyrics/issues/73#issuecomment-1653954768_

**Comment by Real0ver** [comment]
most indo songs are affected

(Secukupnya - Hindia) this song has the broken right to left lyrics but no word by word lyrics
https://github.com/surfbryce/beautiful-lyrics/assets/68629435/05ca520b-6bec-42c5-82b4-f49c4c41fb2d

**Comment by surfbryce** [comment]
Sorry about that guys; I looking into it given some of these songs were fixed in the past and I found out what went wrong here.

So Malay (which is the language being determined here by the language analysis library I use) can be written in Arabic (Jawi) or Latin (Rumi) scripts. It used to be most common to write it in Jawi but as the language modernized it began to be written in Rumi.

The biggest issue is that it's difficult to determine which one is which from a basic computer analysis and so it groups both together as one identifier "zlm" or both as just Malay.

When I searched up which languages in the world were Right-to-Left "zlm" (Malay) appeared as one of them. The issue though is that most songs are written using Rumi instead of Jawi. I assume they went with the Jawi version since it's still officially used in certain parts of the world.

I'm assuming that if it actually were in Arabic script it would be identified as that instead of Malay - so I'm going to remove "zlm" (Malay) from the Right-to-Left languages list since the Rumi script version is very obviously not Right-to-Left.

You will see this change reflected in the next update!
